marienfressinaud/FreshRSS
# Statistics: update Flotr2 to latest release from EnterpriseDB

It contains a few minor fixes for Google's breaking changes in Blink.

See <https://github.com/EnterpriseDB/Flotr2> for the repo.

Cf. <https://github.com/FreshRSS/FreshRSS/pull/3799#issuecomment-905370969>

How to test the feature manually:

1. Load the latest script.
2. Look at the statistics page.
3. Enjoy… no visible difference. But your mouse cursor position should be detected slightly better.

Pull request checklist:
